### ISSUE TITLE
 GO:0005654 | nucleoplasm | IBA with Q5BAB9 , PTN000217420 , FBgn0035640 , Q13257 , WBGene00003161

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#5554]
* **PTHR ID & PTN node:**

mad2

GO:0005654 | nucleoplasm | IBA with Q5BAB9 , PTN000217420 , FBgn0035640 , Q13257 , WBGene00003161

* **Sequences with problematic annotation (ID + gene/protein name):**

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**

I don't know if this is correct for metazoa, but for yeast, the location of action is the kinetochore  (which isn't classed as nucleoplasm). It may localizize to the nucleoplasm but that isn't were it's active?

* Add the label 'high priority' if needed. Generally high priority issues affect a lot of proteins, and annotations are incorrect (as opposed to just imprecise).

**Comment by ValWood** [comment]
same for

GO:0005737 | cytoplasm | IBA with MGI:1860374 , Q13257 , Q5BAB9 , PTN000217420 , locus:2092125 , FBgn0035640 , Q57ZD6
-- | -- | --


[GO:0005737](https://www.pombase.org/term/GO:0005737)	
[cytoplasm](https://www.pombase.org/term/GO:0005737)
[IBA](http://wiki.geneontology.org/index.php/Inferred_from_Biological_aspect_of_Ancestor_(IBA)) with [MGI:1860374](http://www.informatics.jax.org/accession/MGI:1860374) , [Q13257](https://www.uniprot.org/uniprot/Q13257) , [Q5BAB9](https://www.uniprot.org/uniprot/Q5BAB9) , [PTN000217420](http://www.pantherdb.org/panther/lookupId.jsp?id=PTN000217420) , [locus:2092125](http://arabidopsis.org/servlets/TairObject?accession=locus:2092125) , [FBgn0035640](http://flybase.org/reports/FBgn0035640.html) , [Q57ZD6](https://www.uniprot.org/uniprot/Q57ZD6)

**Comment by marcfeuermann** [comment]
There are many different primary CC annotations for this family. I restricted the annotation to kinetochore.
